rfcs/IncrementalDelivery.md

```diff
@@ -17,6 +17,7 @@ The HTTP response for an incrementally delivered response should conform to the
 An example response body will look like:
 
 ```
+
```

Comment:
this here was a wrong move, C# servers doesn't emit preamble CRLF's 

cc @michaelstaib care to verify

Reply:
Good catch! Based on https://github.com/ChilliCream/hotchocolate/blob/main/src/HotChocolate/Core/src/Execution/Serialization/MultiPartResponseStreamFormatter.cs and https://github.com/ChilliCream/hotchocolate/blob/80350d82253419a9ff631bbbaae0f640eb834753/src/HotChocolate/Core/src/Execution/Serialization/MultiPartResponseStreamFormatter.Constants.cs it does in fact appear that Hot Choclate's implementation of multipart streaming is not conformant with RFC 1341 and may not work with clients such as the popular Node `busboy` parser. I've filed #5334. I'm glad you caught this bug now while incremental delivery is still in the RFC stage!